packages/cli-e2e/setup.ts

```diff
@@ -0,0 +1,23 @@
+import type {Browser} from 'puppeteer';
+import {connectToChromeBrowser} from './utils/browser';
+import {clearKeychain, loginWithOffice} from './utils/login';
+
+async function clearChromeBrowsingData(browser: Browser) {
+  const pages = await browser.pages();
+
+  const pageClearDataPromise = [];
+  for (const page of pages) {
```

Comment:
Do we need to do it on all pages 🤔 ?

Reply:
Tbh, I am not sure. I stupidly applied it to all pages just in case. :S